shinyswatch/_get_theme_deps.py

```diff
@@ -33,22 +37,31 @@ def get_theme_deps(name: BSW5_THEME_NAME) -> list[HTMLDependency]:
         # This is to prevent the Shiny bootstrap stylesheet from being loaded and instead load the bootswatch + bootstrap stylesheet
         # _Disable_ bootstrap html dep
         # Prevents  bootstrap from being loaded at a later time (Ex: shiny.ui.card() https://github.com/rstudio/py-shiny/blob/d08af1a8534677c7026b60559cd5eafc5f6608d7/shiny/ui/_navs.py#L983)
+        #
+        # bootstrap_deps_suppress(["css", "js"]),
+        # TODO: Replace the next three lines with the above line when available
+        HTMLDependency(
+            name="bootstrap-js",
+            version=shiny_bootstrap_version + "9999",
+        ),
+        HTMLDependency(
+            name="bootstrap-css",
+            version=shiny_bootstrap_version + "9999",
+        ),
         HTMLDependency(
             name="bootstrap",
-            version=base_dep_version,
+            version=shiny_bootstrap_version + "9999",
         ),
-        # Use a custom version of bootstrap with no stylesheets/JS
-        bs_dep_no_files,
         # Add in the matching JS files
         HTMLDependency(
-            name="bootstrap-js",
+            name=f"bootstrap-{name}-js",
             version=bsw5_version,
             source={"package": "shinyswatch", "subdir": bs5_path},
             script={"src": "bootstrap.bundle.min.js"},
         ),
```

Comment:
We might have a problem with multiple JS bundles being added now. Could we just require https://github.com/posit-dev/py-shiny/pull/1282 and remove this JS dependency?